app/pods/cases/case/subcases/subcase/decision/template.hbs

```diff
@@ -0,0 +1,3 @@
+<Documents::DocumentList
+  @pieces={{this.model}}
+/>
```

Comment:
Deze component hergebruiken, daar ben ik voor te vinden.
Echter willen ze in deze view alles read-only hebben.
Bekijken, downloaden enkel
Kunnen we deze component makkelijk een @readOnly={{true}} meegeven en alle onderliggende componenten ook of maken we beter een reeks nieuwe componenten ?

Reply:
Samen met @ValenberghsSven besproken: we laten read-only componenten voor documenten buiten scope van dit ticket. Nieuwe designs voor meer read-only document-cards zijn in aantocht.